content/clem109/drone-wechat/index.md

```diff
@@ -1,14 +1,14 @@
 ---
 date: 2018-02-19T00:00:00+00:00
-title: Wechat for Work
+title:  企业微信 Enterprise WeChat
```

Comment:
And why this chinese string on an english website when the translation is clear?

Reply:
The vast majority of people who will use this will probably be Chinese, I'm happy to change it back with just the English string for consistency although my assumption is people will search for the Chinese string.

Reply:
English only +1